bundles/org.openhab.binding.ipcamera/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -3074,4 +3075,10 @@
 			</options>
 		</state>
 	</channel-type>
+
+	<channel-type id="autoTracking" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Auto Tracking</label>
+		<description>Turn the automatic mode for the tracking ON and OFF.</description>
```

Comment:
Don;t forget to run the i18n tool after adjusting this description.
```suggestion
		<description>Turn the automatic mode for tracking ON or OFF.</description>
```

Reply:
Thx for the hint. All requested changes are done.